### PR TITLE
Homepage changes

### PR DESCRIPTION
**Pull request by xybrnet**
These changes depend on the new the 'how to use' template in Wazimap.
@longhotsummer

### CODE REVIEW

File: wazimap_za/templates/about_youth.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+        <p>The Western Cape Youth Explorer draws on a number of data sources to profile the situation of young people and the contexts in which they live. Indicators on young people’s education, economic opportunities, family and living environment, health and more are available at a range of geographic levels, from electoral wards to local municipalities and district municipalities. The tool provides simple data visualisations and allows for further interactive exploration of the data.</p>
+
+        <h2 class="subhead">A work in progress</h2>
+
+        <p>So far we have added indicators using data from the 2011 Census, the South African Police Service (SAPS) and the Western Cape Department of Health. During the next few months we will be adding more indicators that draw on administrative data from various government departments, so keep checking the site for updates!</p>
+
+        <h2 class="subhead">Get in touch</h2>
+
+        <p>We are constantly trying to improve this site to make it useful and easy to navigate. Please send any questions, comments or feedback to <a href="mailto:info.pii@uct.ac.za">info.pii@uct.ac.za.</a></p>
```
Comment (longhotsummer) [inline wazimap_za/templates/about_youth.html]: Please include a para censusreporter and wazimap. Even if it's just the blurb from the wazimap.co.za homepage, something like: "Wazimap is based on the amazing Census Reporter and is completely open source. You can build one for your country."

**Comment by longhotsummer** [comment]
I'm reluctant to add a 'using' page to the underlying wazimap just to have it for this site. I don't think it has that much value.

Can we ship this without that page and then discuss it with emily? We could potentially put the content on the homepage.

File: wazimap_za/templates/about_youth.html (hunk 1 above)
Comment (xybrnet) [inline wazimap_za/templates/about_youth.html]: Cool, I also thought it strange not being there.
I've added a paragraph on the about page.

**Comment by xybrnet** [comment]
I've also removed the how to use page for now.

**Pull request merged**